src/App.js

```diff
@@ -3,39 +3,69 @@ import styled, { css } from "styled-components";
 import LoginComponent from "./apps/Login";
 import RegistrationComponent from "./apps/Registration";
 import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
-import firebase from "firebase";
+import firebase from "firebase/app";
 import UserContext from "./contexts/UserContext";
-import RunningProcesses from "./components/RunningProcesses";
+import OfficeView from "./apps/OfficeView";
+import PropTypes from "prop-types";
+import Dashboard from "./apps/Dashboard";
 
 const Background = styled.div`
     ${(props) =>
-        props.centered &&
-        css`
+    props.centered &&
+    css`
             display: flex;
             align-items: center;
             justify-content: center;
             height: 100vh;
         `}
 `;
 
+Background.propTypes = {
+    centered: PropTypes.bool,
+};
+
 function App() {
     const [user, setUser] = useState(null);
 
     useEffect(() => {
-        firebase.auth().onAuthStateChanged(setUser);
+        let unsubscribers = [];
```

Comment:
I think `unsubscribers` can be `const`. A bit nit-picky, but should we consider that?
```suggestion
        const unsubscribers = [];
```

Reply:
Good point @yashika51 can you please fix

Reply:
okay